pkg/golang/mod/parse.go

```diff
@@ -138,3 +153,7 @@ func lessThan117(ver string) bool {
 
 	return major <= 1 && minor < 17
 }
+
+func pkgID(name, version string) string {
+	return fmt.Sprintf("%s@v%s", name, version)
+}
```

Comment:
i am not sure that understand - why do we need `v`version prefix? 
I mean why don't we use this function:
https://github.com/aquasecurity/go-dep-parser/blob/f6e7eca870438a14ea2d477660227cc5298c3a89/pkg/utils/utils.go#L55

Reply:
Added a comment
https://github.com/aquasecurity/go-dep-parser/pull/176/commits/01065334947fdd71efb759ec46cc28eb7cd1f3f1

Go has their ID format like `<module_name>@v<module_version>`. It is better to comply with that so we can easily identify the module path like `$GOPATH/pkg/mod/github.com/aquasecurity/go-dep-parser@v0.0.0-20230130190635-5e31092b0621`. You can see the `v` prefix.